docs/ROLLOUT_STRATEGY_GUIDE.md

```diff
@@ -0,0 +1,364 @@
+# Rollout Strategy Guide
```

Comment:
Can you move this doc to the website repo?

Reply:
yes.